app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -169,24 +172,41 @@ export default function CompleteSignup() {
           <InputLabel className={authClasses.formLabel} htmlFor="birthdate">
             Birthday
           </InputLabel>
-          <TextField
-            className={authClasses.formField}
-            id="birthdate"
-            fullWidth
-            variant="standard"
-            name="birthdate"
-            type="date"
-            InputLabelProps={{
-              shrink: true,
-            }}
-            inputRef={register({
-              required: "Enter your birthdate",
-              validate: (stringDate) =>
-                validatePastDate(stringDate) ||
-                "Must be a valid date in the past.",
-            })}
-            helperText={errors?.birthdate?.message}
-          />
+          <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <Controller
+              {...{ control }}
+              defaultValue={getValues().meetingDT}
```

Comment:
Looks like you're trying to set "dd.mm.rrrr" as a default, but it needs to be a valid date or date string. I would do this (see my other comment for an explanation): 
```suggestion
              defaultValue={new Date().toISOString().split("T")[0]}
```